code/modules/mob/living/carbon/xenomorph/castes/queen/castedatum_queen.dm

```diff
@@ -47,6 +47,22 @@
 	// *** Queen Abilities *** //
 	queen_leader_limit = 1 //Amount of leaders allowed
 
+	actions = list(
+		/datum/action/xeno_action/xeno_resting,
+		/datum/action/xeno_action/regurgitate,
+		/datum/action/xeno_action/plant_weeds,
+		/datum/action/xeno_action/choose_resin,
+		/datum/action/xeno_action/activable/secrete_resin,
+		/datum/action/xeno_action/grow_ovipositor,
+		/datum/action/xeno_action/activable/screech,
+		/datum/action/xeno_action/activable/corrosive_acid/strong,
+		// /datum/action/xeno_action/activable/gut, We're taking this out for now.

```

Comment:
Commented out code

Reply:
That was there before i made any changes, should i still remove it?

Reply:
Please do.